section4/exercises/classMethods.js

```diff
@@ -28,13 +28,23 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+  changeProtein() {
```

Comment:
For each of these methods, we want you to use arguments (so passing in a topping or protein) and then updating the attribute's value. I would highly recommend you understand this and update!